scrapy/item.py

```diff
@@ -96,6 +97,13 @@ def __repr__(self):
     def copy(self):
         return self.__class__(self)
 
+    def deepcopy(self):
+        """Returns a `deep copy`_ of this item.
```

Comment:
Super-minor nitpick: according to [PEP-257](https://www.python.org/dev/peps/pep-0257/), it should be "Return ...", not "Returns ...".

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".